app/components/cases/cases-header.hbs

```diff
@@ -1,36 +1,57 @@
 <Auk::Navbar @skin="gray-100">
-    <Auk::Toolbar @auto={{true}} as |Toolbar|>
-      <Toolbar.Group @position="left" as |Group|>
+  <Auk::Toolbar @auto={{true}} as |Toolbar|>
+    <Toolbar.Group @position="left" as |Group|>
+      <Group.Item>
+        <h1 data-test-cases-header-title class="auk-toolbar__title">
+          {{t "cases"}}
+        </h1>
+      </Group.Item>
+    </Toolbar.Group>
+    <Toolbar.Group @position="right" as |Group|>
+      <Group.Item class="auk-o-flex">
+        {{#if @nameSearchText}}
+          <AuButton
+            @skin="naked"
+            @icon="undo"
+            @hideText={{true}}
+            {{on "click" this.clearFilter}}
+          >
+            {{t "clear-filter"}}
+          </AuButton>
+        {{/if}}
+        <AuInput
+          @width="block"
+          @icon="search"
+          @value={{@nameSearchText}}
```

Comment:
`@value` is going to do a two-way binding of the value, so that `@nameSearchText` gets immediately updated as soon as the input changes, which bypasses the `{{on "input" ... }}`.

We don't notice this because we don't actually use the buffer for anything, what actually causes the search to fire is the `setNameSearchText` method we pass in to this component. We can remove the buffer altogether by removing this two-way binding and using `value={{@nameSearchText}}`. This way the initial value is set, but it doesn't auto-update.

Removing the buffer is also beneficial because otherwise we need to add a `setupController` hook to set the `searchTextNameBuffer` field (otherwise when you e.g. reload the page when a filter is set, the input field won't be pre-filled with the right value because the buffer would be empty). By directly using the query param we don't need to take all of that into account.

Reply:
https://github.com/kanselarij-vlaanderen/frontend-kaleidos/pull/2045/commits/2924ea0abf3285b13c9c406d3899154e1d094482